core/app/models/spree/refund.rb

```diff
@@ -33,22 +30,23 @@ def description
       payment.payment_method.name
     end
 
-    private
-
     # attempts to perform the refund.
     # raises an error if the refund fails.
     def perform!
       return true if transaction_id.present?
 
       credit_cents = Spree::Money.new(amount.to_f, currency: payment.currency).money.cents
 
-      @response = process!(credit_cents)
+      response = process!(credit_cents)
+      log_entries.create!(details: response.to_yaml)
 
-      self.transaction_id = @response.authorization
+      self.transaction_id = response.authorization
```

Comment:
Sorry Jordan, I definitely agree with what you've suggested. 

Had to augment your method suggestion slightly, to be more verbose and to return the refund.